e2e/tests-dfx/sns.bash

```diff
@@ -31,7 +34,16 @@ teardown() {
 
 @test "sns config validate approves a valid configuration" {
     dfx_new
+    cp "${BATS_TEST_DIRNAME}/../assets/sns/valid_sns_init_config.yaml" "$SNS_CONFIG_FILE_NAME"
+    cp "${BATS_TEST_DIRNAME}/../assets/sns/logo.svg" .
```

Comment:
If you rename `assets/sns/valid_sns_init_config.yml` to `assets/sns/sns.yml`, then this would be equivalent:
```suggestion
    install_asset sns
```